src/pattern-matching/destructuring-structs.md

```diff
@@ -5,3 +5,10 @@ You can also destructure `structs`:
 ```rust,editable
 {{#include ../../third_party/rust-by-example/destructuring-structs.rs}}
 ```
+<details>
+  
+Key Points:
+* Showcase the different ways you can match on a struct by giving the fields literal values, variables, and wildcards.
+    * Illustrate how limitating a literal value can be, if it’s unnecessary.
```

Comment:
Hey @cchiw, I don't understand what you mean by this?

Can you make the steps more direct: instead of "showcase the different ways", try saying what the instructor should do in detail.

Reply:
Ok, modified. Let me know what you think